internal/uvm/create_lcow.go

```diff
@@ -261,11 +289,25 @@ func CreateLCOW(opts *OptionsLCOW) (_ *UtilityVM, err error) {
 		kernelArgs += " " + opts.KernelBootOptions
 	}
 
-	// Start GCS with stderr pointing to the vsock port created below in
-	// order to forward guest logs to logrus.
-	initArgs := fmt.Sprintf("/bin/vsockexec -e %d /bin/gcs -log-format json -loglevel %s",
-		linuxLogVsockPort,
-		logrus.StandardLogger().Level.String())
+	// With default options, run GCS with stderr pointing to the vsock port
+	// created below in order to forward guest logs to logrus.
+	initArgs := "/bin/vsockexec"
+
+	if *opts.ForwardStdout {
+		initArgs += fmt.Sprintf(" -o %d", linuxLogVsockPort)
```

Comment:
Can these really reuse the same port with no issues?

Reply:
Yes, vsockexec is smart enough to just dup the fd in that case.